### DISCUSSION TITLE
In AirSim/Source/WorldSimApi.cpp:1051

### DISCUSSION THREAD

**Discussion by YXH6620** [microsoft/AirSim#4482] [Support Q&A]
When I use AirSim as a plugins and compile it came to some errors：
Plugins\AirSim\Source\WorldSimApi.cpp(1009): **error C2653: “AirSimSettings”: 不是类或命名空间名称**
Plugins\AirSim\Source\WorldSimApi.cpp(1009): **error C3861: “singleton”: 找不到标识符**
Plugins\AirSim\Source\WorldSimApi.cpp(1009): **error C2660: “msr::airlib::EarthUtils::nedToGeodetic”: 函数不接受 1 个参数**
Plugins\AirSim\Source\AirLib\include\common/EarthUtils.hpp(291): note: 参见“msr::airlib::EarthUtils::nedToGeodetic”的声明
Plugins\AirSim\Source\WorldSimApi.cpp(1012): error **C3861: “Vector2r”: 找不到标识符**
Plugins\AirSim\Source\WorldSimApi.cpp(1013): error **C3861: “Vector2r”: 找不到标识符**

I have not changed the code....
So what's the matter with it?

**Comment by zimmy87** [comment]
Just from looking at the line numbers, it looks like you're using a version of the plugin that doesn't match the current master. Can you sync to the latest master and then retest?
